packages/elements/src/components/ino-avatar/ino-avatar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Component, ComponentInterface, Prop, h, Element } from '@stencil/core';
+import classNames from 'classnames';
+import { hasSlotContent } from '../../util/component-utils';
+import primary from '../../assets/ino-avatar/primary.svg';
+import secondary from '../../assets/ino-avatar/secondary.svg';
+import primary_dashed from '../../assets/ino-avatar/primary_dashed.svg';
+import secondary_dashed from '../../assets/ino-avatar/secondary_dashed.svg';
+
+/**
+ * @slot icon-slot - For the icon to be displayed in the avatar.
+ */
+@Component({
+  tag: 'ino-avatar',
+  styleUrl: 'ino-avatar.scss',
+  shadow: false,
+})
+export class Avatar implements ComponentInterface {
+  @Element() el: HTMLElement;
+
+  /**
+   * The initials of the avatar.
+   */
+  @Prop() initials: string = '';
+
+  /**
+   * The img of the avatar.
+   */
+  @Prop() src: string = '';
+
+  /**
+   * Flag to enable hover effect.
+   */
+  @Prop() interactive: boolean = false;
+
+  /**
+   * The style variant of the avatar.
+   * Can be 'dashed' | 'solid'.
+   */
+  @Prop() variant: 'dashed' | 'solid' = 'solid';
+
+  /**
+   * Colors the border of the avatar in the secondary color.
+   */
+  @Prop() colorSecondary: boolean = false;
+
+  renderAvatarBorder() {
+    const isDashed = this.variant === 'dashed';
+    const isSecondary = this.colorSecondary;
+
+    // Select appropriate SVG based on variant and color
+    let svgContent;
+    if (isSecondary) {
+      svgContent = isDashed ? secondary_dashed : secondary;
+    } else {
+      svgContent = isDashed ? primary_dashed : primary;
+    }
+    // Decode the base64 string, otherwise only the string value is displayed
+    const decodedSvgContent = window.atob(svgContent.split(',')[1]);
+
+    return <div class="ino-avatar__border" innerHTML={decodedSvgContent} />;
+  }
+  render() {
+    const avatarClasses = classNames({
+      'ino-avatar': true,
+      'ino-avatar--interactive': this.interactive,
+      'ino-avatar--dashed': this.variant === 'dashed',
+      'ino-avatar--solid': this.variant === 'solid',
+    });
+
+    const hasIconSlot = hasSlotContent(this.el, 'icon-slot');
+
+    const avatarBorder = this.renderAvatarBorder();
+
+    return (
+      <div class={avatarClasses}>
+        {avatarBorder}
+        {this.src ? (
+          <div class="ino-avatar__image image">
+            <div
```

Comment:
Due to A11y requirements it is not recommended to use css background images for meaningful image content, because not text alternative may be provided to assitive technologie (refer to https://www.w3.org/TR/WCAG20-TECHS/F3.html).

Is it possible to use native html img element instead?